src/js/actions/AppsActions.js

```diff
@@ -43,6 +43,14 @@ var AppsActions = {
       });
   },
   createApp: function (newAppAttributes) {
+    if (newAppAttributes == null) {
+      AppDispatcher.dispatch({
+        actionType: AppsEvents.CREATE_APP_ERROR,
+        data: {status: 400}
```

Comment:
We shouldn't have to pass a fake response status, imho. Can we have a catch-all `if` condition in the store instead? Consider this patch: 

https://gist.github.com/pierlo-upitup/15bab4a327ebf3061fd2